Foundation/NSURLSession/NSURLSessionTask.swift

```diff
@@ -901,7 +901,7 @@ extension URLSessionTask {
                 NSURLErrorFailingURLStringErrorKey: $0.absoluteString,
                 ]
         }
-        let error = NSError(domain: NSURLErrorDomain, code: errorCode, userInfo: userInfo)
+        let error = URLError(_nsError: NSError(domain: NSURLErrorDomain, code: errorCode, userInfo: userInfo))
```

Comment:
https://github.com/apple/swift-corelibs-foundation/pull/750#issuecomment-272896323: This would work around [SR-2713](https://bugs.swift.org/browse/SR-2713).